geometry_msgs/PoseStamped   initial_pose
---
geometry_msgs/PoseStamped   corrected_pose